spec/models/merchant_spec.rb

```diff
@@ -9,6 +9,11 @@
     it { should validate_presence_of(:name) }
   end
 
+  it 'can build a merchant' do
+    merchant1 = create(:merchant)
+    expect(merchant1.name).to be_a(String)
+  end
```

Comment:
Nice!  I like how you test this.  I can probably add some tests like this on my branch.  You could consider adding an `expect` line to confirm the new object is an instance of `Merchant`.